conf/nginx/symfony4.conf.erb

```diff
@@ -16,6 +16,7 @@ location ~ ^/index\.php(/|$) {
     fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
     fastcgi_param DOCUMENT_ROOT $realpath_root;
     fastcgi_param HTTPS off;
+    fastcgi_param APP_ENV <%= ENV['APP_ENV'] || 'prod' %>;
```

Comment:
I think this goes double with the environment variables:

https://symfony.com/doc/current/setup/web_server_configuration.html#apache-with-php-fpm

Reply:
I don't understand the link you sent me.

But without it, the application does not work without specifying `APP_ENV=prod` here https://my.osc-fr1.scalingo.com/apps/sample-php-symfony5/variables

Reply:
But this is already injected here https://github.com/Scalingo/php-buildpack/pull/122/files#diff-c7b43338c09c3c9afd420fd8b6bb5198R410 (no need of extra var no?)

Reply:
I thought so but apparently not as without this line the application does not work.

Reply:
Quite weird..

Reply:
But ok, it's not harmful anyway